Scripts/RemediationScripts/Remediate-Azure_AppService_DP_Use_Secure_FTP_Deployment.ps1

```diff
@@ -0,0 +1,1132 @@
+        Write-Host $([Constants]::SingleDashLine)
+
+        $userInputforFTPState = @()
+        if (-not $Force)
+            {
+                if($FTPSOnly.IsPresent -eq $true -or $DisableFTP.IsPresent -eq $true)
+                {
+                    Write-Host "FTP State is already provided as input parameter. " -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
+                }
+                else
+                {
+                Write-Host "Do you want to upadate FTP State? " -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
+                $userInput = Read-Host -Prompt "(Y|N)"
+                Write-Host $([Constants]::SingleDashLine)
+                if($userInput -eq "Y")
+                {
+                    #Write-Host "User has provided consent to secure FTP Deployments on the production slot and all non-production slots for all App Services." -ForegroundColor $([Constants]::MessageType.Update)
+                    Write-Host $([Constants]::SingleDashLine)
+                    Write-Host "Please select 1 to set FTP State as FTPSOnly or select 2 to set FTP State Disabled on the production slot and all non-production slots for all App Services" -ForegroundColor $([Constants]::MessageType.Warning) -NoNewline
+                    $userInput = Read-Host -Prompt "(1|2)"
+                    Write-Host $([Constants]::SingleDashLine)
+
+                    if($userInput -eq "1")
+                    {
+                        $userInputforFTPState="1"
+                        Write-Host "FTP State will be set to FTPSOnly on the production slot and all non-production slots for all App Services." -ForegroundColor $([Constants]::MessageType.Update)
+                        Write-Host $([Constants]::SingleDashLine)
+                    }
+                    else
+                    {
+                         if($userInput -eq "2")
+                         {
+                            $userInputforFTPState="2"
+                            Write-Host "FTP State will be set to Disabled on the production slot and all non-production slots for all App Services." -ForegroundColor $([Constants]::MessageType.Update)
+                            Write-Host $([Constants]::SingleDashLine)
+                         }
+                         else
+                         {
+                            Write-Host "User input is not correct. Exiting." -ForegroundColor $([Constants]::MessageType.Update)
+                            Write-Host $([Constants]::SingleDashLine)
+                            return
+                         }
+                    }
+                }
+                else
+                {
+                    Write-Host "FTP Deployments will not be Enabled for any App Service. Exiting..." -ForegroundColor $([Constants]::MessageType.Update)
```

Comment:
Message can be updated to FTP State setting will not be updated for any App Service. Exiting..

Reply:
Message Updated as mentioned above.